bigflow/build/__init__.py

```diff
@@ -0,0 +1,20 @@
+__all__ = []
+
+def setup(**kwargs):
+    """Speficy parameters of Bigflow project. Must me called from `setup.py`
+    """
+    # Laziliy import 'bigflow.dist' as it depends on 'setuptools'
+    import bigflow.build.dist as d
+    return d.setup(**kwargs)
+
+
+# TODO: Remove in v2.0. COMPAT
```

Comment:
?

Reply:
?

Reply:
What info is provided here?

Reply:
This is a marker, which indicates "Someone need to remove this code after v2.0" 
I've changed comment text.